### PR TITLE
Clean up Gravatar component and add to devdocs

### PR DESCRIPTION
**Pull request by bluefuton**
This PR ES6-ifies the `Gravatar` component and adds an example to Devdocs:

http://calypso.localhost:3000/devdocs/design

screen shot 2016-08-18 at 20 58 17

Test live: https://calypso.live/?branch=update/gravatar-cleanup

### CODE REVIEW

File: client/components/gravatar/docs/example.jsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+ * Internal dependencies
+ */
+import Gravatar from 'components/gravatar';
+
+export default React.createClass( {
+
+	displayName: 'Gravatar',
+
+	render() {
+		const user = {
```
Comment (mtias) [inline client/components/gravatar/docs/example.jsx]: I think it may be better to use current user here.

Comment (bluefuton) [inline client/components/gravatar/docs/example.jsx]: As much as I'm sure everyone would have enjoyed admiring @rralian's photo, that's a good shout @mtias. I'll change it now.

Comment (rralian) [inline client/components/gravatar/docs/example.jsx]: I'mma take down photos of you. See how _you_ like it.

**Comment by bluefuton** [comment]
With apologies to Bob, the devdocs now shows the current user's Gravatar @mtias.

**Comment by mtias** [comment]
🚢

**Pull request merged**